### PR TITLE
[zh-cn]Update kubelet-config-file.md

### PR DESCRIPTION
**Pull request by yanrongshi**
Preview:
https://github.com/kubernetes/website/pull/35346/files?short_path=f07bc6e#diff-f07bc6ebbc53c3296a5ef027c7fc0faa2ea86c686769beaf3d05e637b4a0d853

### CODE REVIEW

**Comment by tengqm** [comment]
@yanrongshi Are you paddling the numbers of commits?

File: content/zh-cn/docs/tasks/administer-cluster/kubelet-config-file.md [hunk 1]
```diff
@@ -43,19 +43,44 @@ The configuration file must be a JSON or YAML representation of the parameters
...
+```
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: kubelet-config-1.20
+  namespace: kube-system
+data:
+  kubelet: |
+    apiVersion: kubelet.config.k8s.io/v1beta1
+    kind: KubeletConfiguration
```
Comment (tengqm) [inline content/zh-cn/docs/tasks/administer-cluster/kubelet-config-file.md:55]: 这个代码段为什么要添加到注释里面？

File: content/zh-cn/docs/tasks/administer-cluster/kubelet-config-file.md [hunk 2]
```diff
@@ -43,19 +43,44 @@ The configuration file must be a JSON or YAML representation of the parameters
...
+    address: "192.168.0.8"
+    port: 20250
+    serializeImagePulls: false
+    evictionHard:
+      memory.available:  "200Mi"
+~
```
Comment (tengqm) [inline content/zh-cn/docs/tasks/administer-cluster/kubelet-config-file.md:83]: 83 行怎么回事？

Comment (yanrongshi) [reply content/zh-cn/docs/tasks/administer-cluster/kubelet-config-file.md:83]: Sorry, I will pay attention to it in the future modification. Thank you for reminding me

**Pull request merged**